engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala

```diff
@@ -24,7 +24,7 @@ sealed trait JobDescriptor[K <: CallKey] {
   def locallyQualifiedInputs: CallInputs
 }
 
-final case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
+case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
```

Comment:
Can we all just agree that implicits are a lovely idea in theory but end up just making everything worse?